unique_orchestrator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "unique_orchestrator"
-version = "0.0.2"
+version = "0.0.3"
 description = ""
 authors = [
     "Martin Fadler <martin.fadler@unique.ch>",
```

Comment:
May be name yourself as the author. Martin most definitely did not author this code.